frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -529,6 +542,27 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
                 },
             },
         ],
+        // used to generate a new survey
+        // but all subsequent operations after generation should occur via the surveyLogic
+        newSurvey: [
+            null as Survey | null,
+            {
+                createSurvey: async () => {
+                    const updatedSurvey = {
```

Comment:
Why is this called `updatedSurvey`? Seems like it's just a new survey so why not `survey` or `newSurvey`?

Reply:
makes sense, will change